layouts/shortcodes/eclipsefdn_adopters.html

```diff
@@ -0,0 +1,20 @@
+<!-- 
+  Copyright (c) 2019 Eclipse Foundation, Inc.
+  This program and the accompanying materials are made available under the
+  terms of the Eclipse Public License v. 2.0 which is available at
+  http://www.eclipse.org/legal/epl-2.0.
+  Contributors:
+    Christopher Guindon <chris.guindon@eclipse-foundation.org>
+    
+  SPDX-License-Identifier: EPL-2.0
+--> 
+{{ $container_id := .Get "container_id" | default "" }}
+{{ $class := .Get "class" | default "eclipsefdn-adopters" }}
+{{ $working_group := .Get "working_group" | default .Site.Params.working_group | default .Page.Params.working_group }}
+{{ $ul_classes := .Get "ul_classes" | default "text-center list-inline" }}
+
+  <div id="{{ $container_id }}" class="{{ $class }}"></div>
+  <script src="{{ "assets/js/eclipsefdn.adopters.js" | absURL}}"></script>
+  <script>
+  eclipseFdnAdopters.getWGList({src_root: "https://api.eclipse.org/adopters", working_group: "{{ $working_group }}", ul_classes: "{{ $ul_classes }}"})
+  </script>
```

Comment:
Let me know if there are other custom params I'm missing here... I think that's all we need? 

They can add "id" or change class names to add custom styles